tracing-subscriber/src/fmt/mod.rs

```diff
@@ -357,6 +434,19 @@ where
     }
 }
 
+impl<N, E, F, W> Into<tracing_core::Dispatch> for SubscriberBuilder<N, E, F, W>
+where
+    N: for<'writer> FormatFields<'writer> + 'static,
+    E: FormatEvent<Registry, N> + 'static,
+    W: MakeWriter + 'static,
+    F: layer::Layer<Formatter<N, E, W>> + Send + Sync + 'static,
```

Comment:
Since, when do we need all these thread safety bounds?

Reply:
Always? `Dispatch::new` takes a `Subscriber + Send + Sync + 'static` since the dawn of time. These bounds are required for the `SubscriberBuilder::finish` method.

Reply:
huh never saw that! good point :D